test/integration/targets/prepare_nxos_tests/tasks/main.yml

```diff
@@ -1,12 +1,17 @@
 ---
-- name: Toggle feature nxapi - Enable
+- name: Enable Feature Privilage
   nxos_config:
     lines:
-      - feature nxapi
       - feature privilege
   connection: network_cli
   ignore_errors: yes
 
+- name: Enable Feature NXAPI
+  nxos_nxapi:
+    state: present
+  connection: network_cli
+  ignore_errors: yes
```

Comment:
Does this actually emit errors on any platform? I don't see how this could while the next task doesn't

Reply:
Your right, we don't need `ignore_errors` here.